src/shared/components/oncoprint/DataUtils.ts

```diff
@@ -53,13 +53,21 @@ const protRenderPriority = {
     'down': 0
 };
 
-export type OncoprintMutationType = "missense" | "inframe" | "fusion" | "trunc";
+export type OncoprintMutationType = "missense" | "inframe" | "fusion" | "promoter" | "trunc";
 
-export function getOncoprintMutationType(type:SimplifiedMutationType):OncoprintMutationType {
-    if (type === "missense" || type === "inframe" || type === "fusion") {
-        return type;
+export function getOncoprintMutationType(d:Mutation):OncoprintMutationType {
```

Comment:
would be good to have a comment explaining why "promoter" is special

Reply:
see line 60